### DISCUSSION TITLE
Reason for CURLE_BAD_FUNCTION_ARGUMENT error

### DISCUSSION THREAD

**Discussion by FADEA** [curl/curl#12619] [Q&A]
I have a program based on curl that has been working fine until recently when I encountered a CURLE_BAD_FUNCTION_ARGUMENT error. How can I determine which parameter is causing the error? Thanks in advance!

**Comment by bagder** [comment]
That's a return code from a function call. Can't you just check which function call you did that had that value returned?

**Reply by FADEA** [reply]
I called curl_easy_perform() and got the error code

**Reply by FADEA** [reply]
The version I am using is 8.1.2
